SystemServices/SystemServices.cpp

```diff
@@ -54,9 +54,7 @@
 #include "powerstate.h"
 #endif /* HAS_API_SYSTEM && HAS_API_POWERSTATE */
 
-#ifdef ENABLE_DEVICE_MANUFACTURER_INFO
 #include "mfrMgr.h"
```

Comment:
Checking why this ENABLE_DEVICE_MANUFACTURER_INFO flag is removed. Also don't we need to include SetBootLoaderPattern within this flag since the mfr lib APIs are not available on all platforms.